packages/twenty-front/src/modules/settings/constants/SettingsObjectModel.ts

```diff
@@ -0,0 +1,2 @@
+export const SETTINGS_OBJECT_MODEL_IS_LABEL_SYNCED_WITH_NAME_LABEL_DEFAULT_VALUE =
```

Comment:
Just a thought, but if it's only one one file, it maybe easier to declare it at the top of the file instead of having to create a separate file. Sounds an easier devX to me